crates/plugins/src/store.rs

```diff
@@ -24,6 +24,17 @@ impl PluginStore {
     }
 
     pub fn try_default() -> Result<Self> {
+        if let Ok(brew_prefix) = std::env::var("HOMEBREW_PREFIX") {
```

Comment:
The check for `TEST_PLUGINS_DIRECTORY` should come first, so that tests running on a Mac with Homebrew don't party on the user's real plugins directory.